makefile

```diff
@@ -159,6 +159,7 @@ libflipper: $(X86_TARGET).so | $(BUILD)/include/flipper/.dir
 
 install-libflipper: libflipper
 	$(_v)cp $(BUILD)/$(X86_TARGET)/$(X86_TARGET).so $(PREFIX)/lib/
+	$(_v)ln -sf $(PREFIX)/lib/$(X86_TARGET).so /usr/lib/$(X86_TARGET).so
```

Comment:
This will break Mac so we will have to do a platform detection here.

Reply:
Also, if you have to do this, why not just install to /usr/lib on Linux?

Reply:
It seems to me like `/usr/lib` is where we would install libflipper if we were distributing it as a system package such as a `.deb`, whereas `/usr/local/lib` is where you'd want to install libflipper if you're building it from source. That being said, I'm not sure why @natedooley's distribution doesn't have `/usr/local/lib` included in the ld search path by default.

https://unix.stackexchange.com/a/8658/157043

Reply:
Did we ever get this figured out? Can't merge this now because this will break Mac. Otherwise I will close this PR.